godot-kotlin/godot-library/src/nativeCore/kotlin/godot/core/Godot.kt

```diff
@@ -68,14 +67,15 @@ object Godot {
             )
             languageIndexRef.compareAndSet(languageIndexRef.value, index)
         }
+        TypeManager.registerEngineTypes()
```

Comment:
`TypeManager.registerEngineTypes()` should be called here instead of `init` as it depends on `languageIndex`.